CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -18,7 +18,7 @@ def get_ddb_connection(
 ) -> "botocore.client.DynamoDB":
     ddbclient = ""
     if env == "local":
-        ddbclient = boto3.client("dynamodb", endpoint_url="http://dynamodb:8000/")
+        ddbclient = boto3.client("dynamodb", endpoint_url="http://localstack:4566/")
```

Comment:
The README needs to be updated if we're switching to using localstack for dynamodb.

Specifically I want to run: `aws dynamodb scan --table-name fridge --endpoint-url http://localstack:4566` but get this error: `Could not connect to the endpoint URL: "http://localstack:4566/"`

Reply:
The APIs work though which is good :). Am I pointing to the incorrect port when I try to do a `aws dynamodb scan`?

Reply:
Yes, we need to point to http://**localhost**:4566 when running `aws dynamo scan` commands.
I've fixed port numbers in the README file.
We need to use **localstack** when accessing the localstack instance from within lambda, and **localhost** when accessing it from the host machine. This is because lambda containers are inside cfm-network.